.github/dependabot.yml

```diff
@@ -8,10 +8,7 @@ updates:
     versioning-strategy: "auto"
     allow:
       - dependency-name: "@seamapi/*"
-      - dependency-type: production
     groups:
-      seam:
-        dependency-type: production
-        update-types:
```

Comment:
Can we still limit to minor and patch? We should not auto merge major versions.

Reply:
Only minor and patch updates to @seamapi/* production dependencies should be updated now